flutter/lib/src/screenshot/recorder.dart

```diff
@@ -21,7 +23,13 @@ class ScreenshotRecorder {
   bool warningLogged = false;
 
   ScreenshotRecorder(this.config, this.options) {
-    final maskingConfig = options.experimental.replay.buildMaskingConfig();
+    SentryMaskingConfig maskingConfig;
+    if (config is ScheduledScreenshotRecorderConfig) {
+      maskingConfig = options.experimental.replay.buildMaskingConfig();
+    } else {
+      maskingConfig = options.experimental.screenshot.buildMaskingConfig();
+    }
```

Comment:
Do we need/want to have two distinct configurations? Sounds like it's going to be a pain to configure and is asking for trouble because stuff will be masked in one capture and not in the other.

Reply:
@vaind Good point, I also discussed it with @buenaflor. 
What about, having global settings that apply for both, and have override options for `SR` and `SS` for power users? Or should we start with the one for both and wait until the community asks for a different behavior?

Reply:
> Or should we start with the one for both and wait until the community asks for a different behavior?

IMO, yes, keep it simple